src/apps/material/material.dev.tsx

```diff
@@ -82,6 +82,16 @@ export default {
       defaultValue: "Reserve",
       control: { type: "text" }
     },
+    reserveFromAnotherLibraryText: {
+      name: "Reserve from another library",
+      defaultValue: "Reserve @materialType elsewhere",
+      control: { type: "text" }
+    },
+    reserveFromAnotherLibraryWithoutMaterialTypeText: {
+      name: "Reserve from another library",
```

Comment:
This is probably a detail. But the Storybook controls will have to labels with the same text now that you have to name properties with the same value, tight?